safe-read.lisp

```diff
@@ -91,7 +95,9 @@
 ;; Handler-case and macro-wrapper for safe reading
 (defmacro safe-read-handler-case (&body body)
   (let ((gensym (gensym)))
-    `(with-temp-package
+    `(with-temp-package ,@(if (eq (first body) :use-list)
+			      (prog1 (list :use-list (second body))
```

Comment:
The indentation looks like crap here, but it looks fine in Emacs. I assume that this is due to Emacs Lisp-Mode using real tabs and GitHub not expanding them correctly.

Reply:
`M-x untabify` will do the job.